packages/xo-server-sdn-controller/src/index.js

```diff
@@ -622,8 +641,21 @@ class SDNController extends EventEmitter {
     vni,
     encrypted,
     mtu,
+    preferredCenterId,
   }) {
-    const privateNetwork = new PrivateNetwork(this, uuidv4())
+    let preferredCenter
+    if (preferredCenterId !== undefined) {
+      preferredCenter = this._xo.getXapiObject(
+        this._xo.getObject(preferredCenterId, 'host')
+      )
+
+      // Put pool of preferred center first
+      const i = poolIds.indexOf(preferredCenterId)
```

Comment:
```suggestion
      const i = poolIds.indexOf(preferredCenterId)
      assert.notStrictEqual(i, -1)
```

Reply:
If `preferredCenterId !== undefined` then it is in `poolIds`.

Reply:
That's the goal of the assert, to make sure of this.